Error while checking build/O2/o2-dataflow for 8886a93bb3d2c95f94b7b6bbba23fd9283ca5b15 at 2022-10-07 22:54:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 434
100% tests passed, 0 tests failed out of 102


## sw/BUILD/QualityControl-latest/log
27/38 Test #33: multinode_test ..........................***Failed   35.73 sec
[3851:qc-check-TST-MultiNodeLocalTest]: 2022-10-07 20:53:08.484142 !!! [1;31mError - Could not find the DPL InfoLogger.[0m
[3852:qc-check-TST-MultiNodeRemoteTest]: 2022-10-07 20:53:08.484142 !!! [1;31mError - Could not find the DPL InfoLogger.[0m
[3862:qc-task-TST-MultiNodeLocalTest40252]: 2022-10-07 20:53:08.513076 !!! [1;31mError - Could not find the DPL InfoLogger[0m
[3850:qc-task-TST-MultiNodeRemoteTest48518]: 2022-10-07 20:53:08.493449 !!! [1;31mError - Could not find the DPL InfoLogger[0m
[ERROR] websocket_server_callback: Error while reading from websocketconnection reset by peer
[ERROR] SEVERE: Device internal-dpl-clock (3846) returned with 128
97% tests passed, 1 tests failed out of 38
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10026/8886a93bb3d2c95f94b7b6bbba23fd9283ca5b15/build_O2_o2-dataflow/pretty.html).

--- a/Detectors/ITSMFT/ITS/workflow/include/ITSWorkflow/DCSParserSpec.h
+++ b/Detectors/ITSMFT/ITS/workflow/include/ITSWorkflow/DCSParserSpec.h
@@ -85,9 +85,6 @@ class ITSDCSParser : public Task
 
   std::string mSelfName = "";
 
-  // Keep track of whether the endOfStream() or stop() has been called
-  bool mStopped = false;
-
   // Whether to use verbose output
   bool mVerboseOutput = false;
 

--- a/Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx
+++ b/Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx
@@ -400,7 +400,7 @@ void ITSDCSParser::pushToCCDB(ProcessingContext& pc)
   std::map<std::string, std::string> metadata;
   std::map<std::string, std::string> headers = api.retrieveHeaders(
     "RCT/Info/RunInformation", metadata, this->mRunNumber);
-  if (headers.empty()) {  // No CCDB entry is found
+  if (headers.empty()) { // No CCDB entry is found
     LOG(error) << "Failed to retrieve headers from CCDB with run number " << this->mRunNumber
                << "\nWill default to using the current time for timestamp information";
     tstart = o2::ccdb::getCurrentTimestamp();

--- a/Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx
+++ b/Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx
@@ -76,6 +76,11 @@ void ITSDCSParser::run(ProcessingContext& pc)
     this->mConfigDCS.clear();
   }
 
+  // Reset saved information for the next EOR file
+  this->mRunNumber = UNSET_INT;
+  this->mConfigVersion = UNSET_INT;
+  this->mRunType = UNSET_SHORT;
+
   return;
 }
 
@@ -204,7 +209,7 @@ void ITSDCSParser::updateAndCheck(int& memValue, const int newValue)
     memValue = newValue;
   } else if (memValue != newValue) {
     // Different value received than the one saved in memory -- throw error
-    throw newValue;
+    throw std::runtime_error(fmt::format("New value {} differs from old value {}", newValue, memValue));
   }
 
   return;
@@ -219,7 +224,7 @@ void ITSDCSParser::updateAndCheck(short int& memValue, const short int newValue)
     memValue = newValue;
   } else if (memValue != newValue) {
     // Different value received than the one saved in memory -- throw error
-    throw newValue;
+    throw std::runtime_error(fmt::format("New value {} differs from old value {}", newValue, memValue));
   }
 
   return;

